cli/lesspass/visual_fingerprint.py

```diff
@@ -0,0 +1,281 @@
+import hmac
+import hashlib
+import sys
+import os
+import random
+import tty
+import termios
+import threading
+
+if os.name == "nt":
+    import msvcrt
+
+
+def user_has_icons_in_terminal():
+    return os.path.exists(os.path.expanduser("~/.fonts/icons-in-terminal.ttf"))
```

Comment:
We will need some documentation and either define limited targets platform or hardening